guacamole/src/main/webapp/translations/ru.json

```diff
@@ -327,9 +327,11 @@
         "FIELD_OPTION_SERVER_LAYOUT_FR_FR_AZERTY" : "French (Azerty)",
         "FIELD_OPTION_SERVER_LAYOUT_IT_IT_QWERTY" : "Italian (Qwerty)",
         "FIELD_OPTION_SERVER_LAYOUT_SV_SE_QWERTY" : "Swedish (Qwerty)",
-	      "FIELD_OPTION_SERVER_LAYOUT_RU_RU_QWERTY" : "Russian (Qwerty)",
+	    "FIELD_OPTION_SERVER_LAYOUT_RU_RU_QWERTY" : "Russian (Qwerty)",
```

Comment:
Looks like a stray tab made it in here...

Reply:
Yep, apparently some time ago via 8d33092c357b62da8b181046e8bb9506820ae885 when the Russian translation was contributed. I'll correct it.